docs/docs/getting-started.md

```diff
@@ -37,13 +37,68 @@ $ browserify -t [ babelify --presets [ react ] ] main.js -o bundle.js
 To install React DOM and build your bundle with webpack:  
 
 ```sh
-$ npm install --save react react-dom babel-preset-react
+$ npm install --save react react-dom babel-preset-react babel-preset-es2015
 $ webpack
 ```
 
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
+A good starting webpack config for production mode would be:
+```javascript
```

Comment:
We'll need to make sure we have whitespace surrounding code fences otherwise this gets dropped inline (I encourage you to run our site locally to ensure it looks right - directions are in the readme in the docs folder).